PRIVACY.md

```diff
@@ -0,0 +1,111 @@
+# Privacy Policy
+Last updated: 2023-08-24
+
+Your privacy is a fundamental right JabRef e.V. respects and supports.
+By using JabRef and its related online services, you choose to share some of this information.
+In this Privacy Policy We explain how We collect, use, and share information about you, along with the choices you have.
```

Comment:
We, Us and Privacy Policy should follow standard English capitalization rules (or at least be consistently written)

Reply:
I believe "Privacy Policy" is a fixed term and a name, so I uppercased it in the document, but I'm not sure.